lsp-mode.el

```diff
@@ -650,6 +650,26 @@ If this is set to nil, `eldoc' will show only the symbol information."
   :type 'boolean
   :group 'lsp-mode)
 
+(defcustom lsp-modeline-code-actions-enable nil
```

Comment:
Let's have this enabled by default and eventually coordinate with lsp-ui-sideline.